daemon/gluster-blockd.c

```diff
@@ -587,7 +587,7 @@ main (int argc, char **argv)
   int wstatus;
 
 
-  if(initLogging()) {
+  if(initLogging("mgmt")) {
```

Comment:
Same here:
 gbProcessCtx gbCtx = GB_DAEMON_MODE; /* set process mode */ ?

Intead of passing the argument ?

Reply:
Will fix it.
Thanks.